src/main/kotlin/no/nav/syfo/infrastructure/database/repository/SenOppfolgingRepository.kt

```diff
@@ -59,18 +59,32 @@ class SenOppfolgingRepository(private val database: DatabaseInterface) : ISenOpp
 
     override fun getUnpublishedKandidater(): List<SenOppfolgingKandidat> =
         database.connection.use { connection ->
-            connection.prepareStatement(GET_UNPUBLISHED).use {
+            connection.prepareStatement(GET_UNPUBLISHED_KANDIDAT).use {
                 it.executeQuery().toList { toPSenOppfolgingKandidat() }
             }.map {
                 it.toSenOppfolgingKandidat(connection.getVurderinger(it.id))
             }
         }
 
-    override fun setPublished(kandidatUuid: UUID) =
+    override fun getUnpublishedVurderinger(): List<Pair<SenOppfolgingKandidat, SenOppfolgingVurdering>> {
```

Comment:
Hvis vi hadde fått til å hente bare `SenOppfolgingKandidat` så kunne alt av kode i service og kafka producer blitt enklere. I så fall så måtte vi vel bare publisert den sist gjorte vurderingen på kandidaten🤔 Hva tenker du?
```suggestion
    override fun getKandidaterWithUnpublishedVurderinger(): List<SenOppfolgingKandidat> {
```

Reply:
Mhm, forsøker noe sånt, tror det blir bedre.